src/main/resources/db/migration/V1__create_tables.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TYPE PHONE_TYPE as ENUM ('home', 'work', 'mobile');
+
+CREATE TYPE ROLE as ENUM ('associate', 'partner', 'senior');
+
+CREATE TYPE STATUS as ENUM ('new', 'assigned', 'on_hold',
+    'approved', 'canceled', 'declined');
+
+CREATE TABLE users
+(
+    id               bigserial
+        constraint PK_users PRIMARY KEY,
+    email            text not null
+        constraint UQ_users_email UNIQUE ,
+    cognito_username text not null
+        constraint UQ_users_cognito_username UNIQUE
+);
+
+CREATE TABLE advisors
+(
+    id   bigserial
```

Comment:
What will happen if a user with such an id has already been created in the applicants table. Can one user be an applicant and an advisor at the same time?